docs/modules/ROOT/pages/release-notes/release-notes-2021-3.adoc

```diff
@@ -0,0 +1,37 @@
+= Payara Community 5.2021.3 Release Notes
+
+== Supported APIs and Applications
+
+* Jakarta EE 8
+* Java EE 8 Applications
+* MicroProfile 4.0
+
+== Notes
+
+=== Hazelcast 4.2 Upgrade
+
+It is important to note in this release that with the upgrade of Hazelcast to version 4.2, support for rolling updates from previous versions of Payara Community Edition to 5.2021.3 will not be supported.
+
```

Comment:
Please add

=== Metro (JAX-WS implementation) remote code vulnerability

We have fixed a remote code vulnerability in the Metro framework. If you have an application deployed on the Payara Server that makes use of the JAX-WS features, please update your environment.